.github/workflows/release.yml

```diff
@@ -97,6 +112,14 @@ jobs:
         run: task build
         if: matrix.operating-system == 'macos-10.15'
 
+      - name: Create autoupdate files
+        run: ${GOPATH}/bin/go-selfupdate arduino-create-agent${{ matrix.ext }} ${TAG_VERSION} # this will create `public/` dir with compressed full bin (<version>/<os>-<arch>.gz) and a json file
+        if: steps.prerelease.outputs.IS_PRE != 'true'
+
+      - name: Upload autoupdate files to Arduino downloads servers
+        run: aws s3 sync public/ s3://${{ secrets.DOWNLOADS_BUCKET }}${{ env.TARGET }} --include "*"
```

Comment:
the upload target should be correct: `env.TARGET = "/CreateBridge/"`, so the full path will be:
http://downloads.arduino.cc/CreateBridge/

So pushing all the files inside `public` should result in:
http://downloads.arduino.cc/CreateBridge/linux-amd64.json
http://downloads.arduino.cc/CreateBridge/1.2.0/linux-amd64.gz